hub/config/src/main/java/uk/gov/ida/hub/config/ConfigModule.java

```diff
@@ -73,7 +74,6 @@ protected void configure() {
         bind(new TypeLiteral<ConfigEntityDataRepository<CountriesConfigEntityData>>(){}).asEagerSingleton();
         bind(new TypeLiteral<ConfigEntityDataRepository<MatchingServiceConfigEntityData>>(){}).asEagerSingleton();
         bind(new TypeLiteral<ConfigEntityDataRepository<IdentityProviderConfigEntityData>>(){}).asEagerSingleton();
-        bind(ObjectMapper.class).toInstance(new ObjectMapper().registerModule(new GuavaModule()));
```

Comment:
D'you know what the `new GuavaModule()` was for?

Reply:
I think GuavaModule is used for deserialising and serialising Optional.absent().

This line creates a new instance of ObjectMapper and it does not have GuavaModule registered. This instance is not from DropWizard. 

This pull request changes Config application to use the existing instance of ObjectMapper provided by DropWizard (created in bootstrap) and this instance already has GuavaModule registered. Please see https://github.com/dropwizard/dropwizard/blob/fb58944deddfa2925adb2fd4a3b4c3dba5998cd7/dropwizard-jackson/src/main/java/io/dropwizard/jackson/Jackson.java#L55.

Reply:
`This line creates a new instance of ObjectMapper and it does not have GuavaModule registered. This instance is not from DropWizard.` should be read as `This line creates a new instance of ObjectMapper and it does not have GuavaModule registered (i.e. it has to manually register GuavaModule) in order to make it work. This instance is not from Dropwizard.`

Reply:
Ok, sounds good to me.

Reply:
👍